src/promptflow/promptflow/azure/operations/_arm_connection_operations.py

```diff
@@ -41,6 +44,11 @@ class ConnectionCategory:
     BingLLMSearch = "BingLLMSearch"
 
 
+class ConnectionAuthType:
+    ApiKey = "ApiKey"
+    AAD = "AAD"
```

Comment:
why AAD here and meid_token previously?

Reply:
AAD here is one of the connection AuthType workspace RP returned to us, 'auth_mode': 'meid_token' is the customer facing value in connection yaml and entity.